### PR TITLE
[Doc] Add English tutorial

### PR DESCRIPTION
**Pull request by Thudjr**
add english docs

### CODE REVIEW

File: docs/api_example_EN.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+# API application case
+
+Through this tutorial, you will quickly learn the API calls of PaddleSeg, and easily train, evaluate and predict semantic segmentation models. We will take BiSeNetV2 and optic disc segmentation datasets as examples to teach you step by step how to call the API to build models, datasets, loss functions, optimizers and other modules.
+
+If you are more interested in the configuration call method, you can refer to[10分钟上手PaddleSeg](https://aistudio.baidu.com/aistudio/projectdetail/1672610)tutorial。
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: If you are more interested in the configuration call method, you can refer to the [tutorial](https://aistudio.baidu.com/aistudio/projectdetail/1672610).

File: docs/api_example_EN.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
+# API application case
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: api_example.md for English doc, api_example_cn.md for Chinese doc.

File: docs/api_example_EN.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+Through this tutorial, you will quickly learn the API calls of PaddleSeg, and easily train, evaluate and predict semantic segmentation models. We will take BiSeNetV2 and optic disc segmentation datasets as examples to teach you step by step how to call the API to build models, datasets, loss functions, optimizers and other modules.
+
+If you are more interested in the configuration call method, you can refer to[10分钟上手PaddleSeg](https://aistudio.baidu.com/aistudio/projectdetail/1672610)tutorial。
+
+**Note**：Please fork the latest version of this project on AI studio, and then run it.
+
+**Note**：For more details of PaddleSeg API，please refer to[API文档](https://github.com/PaddlePaddle/PaddleSeg/tree/develop/docs/apis)
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: For more details of PaddleSeg API，please refer to the [tutorial](https://github.com/PaddlePaddle/PaddleSeg/tree/develop/docs/apis).

File: docs/api_example_EN.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+## Installation and environment configuration of PaddleSeg
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: Only "## Installation"

File: docs/api_example_EN.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+**Note**：For more details of PaddleSeg API，please refer to[API文档](https://github.com/PaddlePaddle/PaddleSeg/tree/develop/docs/apis)
+
+## Installation and environment configuration of PaddleSeg 
+
+```python
+!pip install paddleseg
+```
+
+## Model training
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: Model Training

File: docs/api_example_EN.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+
+```python
+from paddleseg.models import BiSeNetV2
+model = BiSeNetV2(num_classes=2,
+                 lambd=0.25,
+                 align_corners=False,
+                 pretrained=None)
+```
+
+### 2. Load model parameters
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: Load pretrained model

File: docs/api_example_EN.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+# bulid validation dataset
+from paddleseg.datasets import OpticDiscSeg
+val_dataset = OpticDiscSeg(
+    dataset_root='data/optic_disc_seg',
+    transforms=transforms,
+    mode='val'
+)
+```
+
+### 4. Evaluating
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: Evaluation

File: docs/api_example_EN.md [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+
+    if len(image_list) == 0:
+        raise RuntimeError('There are not image file in `--image_path`')
+
+    return image_list, image_dir
+image_path = 'data/optic_disc_seg/JPEGImages/N0010.jpg' # 也可以输入一个包含图像的目录
+image_list, image_dir = get_image_list('data/optic_disc_seg/JPEGImages/N0010.jpg')
+```
+
+### 4. Predict
```
Comment (michaelowenliu) [inline docs/api_example_EN.md]: Prediction

**Review by michaelowenliu — CHANGES_REQUESTED** [review]

File: docs/api_example_EN.md (hunk 1 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 2 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 3 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 5 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 6 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 4 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 7 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

File: docs/api_example_EN.md (hunk 8 above)
Comment (Thudjr) [reply docs/api_example_EN.md]: done

**Review by michaelowenliu — APPROVED** [review]

**Pull request closed without merging**